api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -196,7 +196,10 @@
 			<column name="diagnosis_id" type="int" autoIncrement="true">
 				<constraints primaryKey="true" nullable="false" />
 			</column>
+			<column name="condition_coded" type="int" />
```

Comment:
Shouldn't this be diagnosis instead of condition?

Reply:
Yeah, it should be diagnosis. Changing it appropriately.